bibblio.py

```diff
@@ -237,32 +249,21 @@ def process_item(self, identifier, force=False):
             self._db, Identifier.BIBBLIO_CONTENT_ITEM_ID, content_item_id
         )
 
-        identifier.equivalent_to(
-            self.output_source, bibblio_identifier, 1
-        )
-        return identifier
-
-    def add_coverage_record_for(self, item):
-        """Adds an appropriate CoverageRecord for this particular
-        identifier and any equivalent identifiers.
-
-        Because Bibblio is text-based recommendation source, it doesn't
-        make sense to represent the same work multiple times.
-        """
-        equivalent_identifier_ids = item.equivalent_identifier_ids()[item.id]
-
-        equivalent_identifiers = self._db.query(Identifier).filter(
-            Identifier.id.in_(equivalent_identifier_ids)
+        work_identifiers = self._db.query(Identifier).filter(
+            Identifier.id.in_(work.all_identifier_ids())
         )
-        for identifier in equivalent_identifiers:
-            super(BibblioCoverageProvider, self).add_coverage_record_for(identifier)
+        for identifier in work_identifiers:
+            identifier.equivalent_to(
+                self.output_source, bibblio_identifier, 1
```

Comment:
I'm not comfortable with a certainty level of 1 here. By default, `Work.all_identifier_ids` finds all Identifiers where our confidence is 0.5 or higher. If you're only 50% confident that an Identifier is part of a Work, you should only be 50% confident that that Identifier is equivalent to an Identifier you found by looking up the Work. 

Ideally you would call equivalent_to() with an appropriate confidence value for each Identifier. But we don't have that information. For performance reasons it's handled by a SQL function.

Fortunately, I don't think you need to do this at all anymore. The code you had before created extra CoverageRecords (some of which probably shouldn't have been created) to avoid doing duplicate work. Now that this is a WorkCoverageRecord, the work is being properly tracked. Unless you're using this equivalence for something else I don't think we need to store this information at all.